### PR TITLE
Properly defined maximum and minimum values

### LINKED ISSUE #1402: Define maximumValue and minimumValue properly
As of #1399,  `maximumValue` and `minimumValue` are defined on `Long`, and they are defined on the `int64_t` sizes, as it should be.

We need to change the values of `Int minimumValue` etc. to conform to the exact platform-independent sizes.

We also need to add these properties to the unsigned versions.

### PR DESCRIPTION
**Pull request by marcusnaslund**
Fixes #1402

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Peer review @thomasfanell ? Slightly more exciting code than last time. :)

File: source/system/math.ooc [hunk 1]
```diff
@@ -42,11 +35,6 @@ extend ULong {
 }
 
 extend Int {
-	negativeInfinity ::= static INT_MIN
-	positiveInfinity ::= static INT_MAX
-	epsilon ::= static 1
```
Comment (marcusnaslund) [inline source/system/math.ooc]: `Int epsilon` just kind of makes no sense.

**Comment by marcusnaslund** [comment]
@thomasfanell Magic fails at

`slice: func (start: Int, distance := Int maximumValue) -> This`

with the message

`[107, 58].....redundant type information, reduce to (start, distance, maximumValue: Int)`

**Comment by thomasfanell** [comment]
@marcusnaslund a fix is on the way

**Pull request merged**